app/code/Magento/Wishlist/Model/Rss/Wishlist.php

```diff
@@ -165,7 +163,7 @@ public function getRssData()
                 }
                 $description .= '</p>';
 
-                if (trim($product->getDescription()) != '') {
+                if ($product->getDescription() && trim($product->getDescription()) != '') {
```

Comment:
```suggestion
                if (is_string($product->getDescription()) && trim($product->getDescription()) != '') {
```

Reply:
@ihor-sviziev I added this changes, thank you.